action.yml

```diff
@@ -396,3 +400,4 @@ runs:
           fi
         fi
         python -m nuitka $ARGS "${{ inputs.script-name }}"
+      working-directory: ${{ inputs.working-directory }}
```

Comment:
Am I correct in seeing that this doesn't cover the Windows branch, which of course has no right to exist, since all platform shell script should be merged, with Windows also using `bash`. But if you added it to the Windows branch too,, I think that would be sufficient for now.

Reply:
Correct. I have no experience on using Github action on Windows.
I added for windows.